src/OAuth2/Storage/Pdo.php

```diff
@@ -317,17 +335,39 @@ public function getUser($username)
             'user_id' => $username
         ), $userInfo);
     }
+    
+    public function hashPassword($password)
```

Comment:
I'm comfortable with leaving that method accessible from outside of the object but equally happy to change that if it's a blocker to moving forward with the change.